src/if_py_both.h

```diff
@@ -412,7 +412,11 @@ write_output(OutputObject *self, PyObject *string)
 
     Py_BEGIN_ALLOW_THREADS
     Python_Lock_Vim();
+    int save_emsg_severe = emsg_severe;
+    if (error)
+	emsg_severe = TRUE;
     writer((writefn)(error ? emsg : msg), (char_u *)str, len);
+    emsg_severe = save_emsg_severe;
```

Comment:
I'm not 100% sure that this save/restore is necessary or correct. emsg_core actually forces emesg_severe back to FALSE, so maybe we don't want this.